### PR TITLE
Optimize postgres configuration

### PR DESCRIPTION
**Pull request by jacobdgm**
This PR changes the Postgres configuration file, `postgresql.config`, so that it hopefully runs better on our staging and production servers. It also changes the docker setup so that this file ends up in the right place in the postgres container.

Requesting @kqct's review, to ensure all the Docker-related changes are sensible, and requesting @ahankinson's review to ensure all the Postgres-related changes are reasonable.

Testing locally, these changes appear to have a slight, but hardly significant significant, positive effect on query evaluation times. We expect it to have a larger effect when running on the DRAC servers, since the configuration is meant to match the specs of those VMs and not those of my local computer.

### CODE REVIEW

File: postgres/postgresql.conf [hunk 1]
```diff
@@ -0,0 +1,822 @@
...
+#ssl_passphrase_command_supports_reload = off
+
+
+#------------------------------------------------------------------------------
+# RESOURCE USAGE (except WAL)
+#------------------------------------------------------------------------------
+
+# - Memory -
+
+shared_buffers = 128MB			# min 128kB
```
Comment (ahankinson) [inline postgres/postgresql.conf:155]: This is not correct. You need to check these values against the PG config generator. I'll post what I get from that in a comment so it doesn't get lost here, but basically this setting, and the cache size settings are the most important ones.

So this should be *at least* 1GB, if not 2.

**Comment by ahankinson** [comment]
Here's what I get from pgtune:

```
# DB Version: 14
# OS Type: linux
# DB Type: web
# Total Memory (RAM): 8 GB
# CPUs num: 8
# Connections num: 100
# Data Storage: hdd

max_connections = 100
shared_buffers = 2GB
effective_cache_size = 6GB
maintenance_work_mem = 512MB
checkpoint_completion_target = 0.9
wal_buffers = 16MB
default_statistics_target = 100
random_page_cost = 4
effective_io_concurrency = 2
work_mem = 5242kB
min_wal_size = 1GB
max_wal_size = 4GB
max_worker_processes = 8
max_parallel_workers_per_gather = 4
max_parallel_workers = 8
max_parallel_maintenance_workers = 4
```

File: postgres/postgresql.conf (hunk 1 above)
Comment (jacobdgm) [reply postgres/postgresql.conf:155]: ah - thanks for catching this! I thought all the lines below what I copied from pgtune were commented out, but I see this overwrites the 2GB specified near the top of the file.

Comment (ahankinson) [reply postgres/postgresql.conf:155]: You should clean this file up. Any line that is commented out is anyway the default setting, so you don't lose anything by just deleting those lines.

**Comment by jacobdgm** [comment]
> ```
> # DB Version: 14
> # OS Type: linux
> # DB Type: web
> # Total Memory (RAM): 8 GB
> # CPUs num: 8
> # Connections num: 100
> # Data Storage: hdd
> ```

Thank you for this - evidently, I guessed wrong for Data Storage, and didn't know what to use for Connections num. I'll use the values from your comment.

**Comment by jacobdgm** [comment]
Github is being ornery - closing this; going to reopen and see if my most recently pushed changes will appear.

**Pull request closed without merging**